.github/PULL_REQUEST_TEMPLATE_nolabels

```diff
@@ -0,0 +1,13 @@
+## ATTENTION - Release Note ack required!
+
+Update the below named `release-note-needed` block to one of:
+
+**\`\`\`release-note**
+**\`\`\`release-note-action-required**
+**\`\`\`release-note-none**
+
+Then describe your change as a summarized release note within the block.
+
+```release-note-needed
+...release-note block...
```

Comment:
If we want to be able to take from the title, that needs to be independent of the category (default, action required, experimental features). So, I think it needs to go in the body: `TITLE`.